src/lib/select/select.spec.ts

```diff
@@ -1800,6 +1800,31 @@ describe('MdSelect', () => {
         expect(options[2].getAttribute('aria-selected')).toEqual('false');
       });
 
+      it('should show ripples for options by default', () => {
+        expect(overlayContainerElement.querySelectorAll('.mat-ripple-element').length)
+          .toBe(0, 'Expected no ripples to show up initially.');
+
+        dispatchFakeEvent(options[0], 'mousedown');
+        dispatchFakeEvent(options[0], 'mouseup');
+
+        expect(overlayContainerElement.querySelectorAll('.mat-ripple-element').length)
+          .toBe(1, 'Expected one ripple to show up after a fake click.');
+      });
+
+      it('should not show ripples for options if disableRipple is set to true', () => {
+        fixture.componentInstance.disableRipple = true;
+        fixture.detectChanges();
+
+        expect(overlayContainerElement.querySelectorAll('.mat-ripple-element').length)
+          .toBe(0, 'Expected no ripples to show up initially.');
+
+        dispatchFakeEvent(options[0], 'mousedown');
```

Comment:
I'm not sure whether we care about the DOM events that are happening in these tests since this is tested well enough in the ripple tests. Looping through all the options and verifying that the flag is correct should be enough. Also it could use a test where it adds a new option and check that it got the proper `disableRipple` value.